sonar-ws/src/main/protobuf/ws-permissions.proto

```diff
@@ -81,3 +81,20 @@ message SearchProjectPermissionsResponse {
   repeated Permission permissions = 2;
   optional sonarqube.ws.commons.Paging paging = 3;
 }
+
+message PermissionTemplate {
+  optional string key = 1;
+  optional string name = 2;
+  optional string description = 3;
+  optional string projectPattern = 4;
+  optional string createdAt = 5;
+  optional string updatedAt = 6;
```

Comment:
I know that it's not done on other proto files, but these 2 dates deserve to be commented to give the format of strings.